src/components/DatePickerRange.react.js

```diff
@@ -263,6 +264,31 @@ DatePickerRange.propTypes = {
     persistence_type: PropTypes.oneOf(['local', 'session', 'memory']),
 };
 
+DatePickerRange.persistenceTransforms = {
+    end_date: {
+        extract: propValue => {
+            if (!(propValue === null || propValue === undefined)) {
```

Comment:
This check is fine but can be simplified by using https://ramdajs.com/docs/#isNil instead

Reply:
57e5019